templates/email/__roomInfo.html.twig

```diff
@@ -10,7 +10,7 @@
     <p>
         {% trans with {'{email}': user.email,'{uid}': room.uid} %}
             Folgende Daten benötigen Sie um der Konferenz beizutreten:<br>
-            Ihre Email Adresse: {email}<br>
+            Ihre E-Mail-Adresse: {email}<br>
```

Comment:
Hallo Alex,
wenn du einen Text änderst, ist es auch notwendig den Text in dem Ordner translation/messages+intl-icu.en und fr anpassen.
Das ist dafür da, da wir hier schon eine Übersetzung vorbereitet haben. und nur noch auf Übersetzer warten.
Da muss irgendwo ein ganzer Textblock sein von {%trans%] bis {%endtrans%} den kannst du mir suchen finden :)
und da einfach den Text so anpassen, wie du ihn hier geschrieben hast.